ci-operator/config/stackrox/collector/stackrox-collector-mauro-setup-cpaas-support-packages.yaml

```diff
@@ -849,7 +864,7 @@ tests:
     test:
     - as: push-drivers
       commands: |
-        /scripts/push-drivers.sh
+        /scripts/push-drivers.sh gs://collector-modules-osci
```

Comment:
It's probably not related to the change, but I've missed this part. Would you remind me pls, why is it needed to call `push-drivers.sh` with a test and a post-submit?

Reply:
This needs to run in a test so that we can provide credentials, thus preventing just anyone to push garbage onto the GCP bucket.

Reply:
As for why is it needed on both pre and post submits, it's because sometimes we need to rebuild the drivers on PRs and this helps with testing them.